src/core/unit.c

```diff
@@ -3453,6 +3453,7 @@ int unit_patch_contexts(Unit *u) {
 
                         ec->private_tmp = true;
                         ec->remove_ipc = true;
+                        ec->private_devices = true;
```

Comment:
hmm, why should we imply PrivateDevices= here, care to elaborate? and where do the docs suggest this was already the case?